tsp-output/@azure-tools/typespec-csharp/azoai/AzureOpenAI/Models/Input/AzureCreateChatCompletionRequest.cs

```diff
@@ -0,0 +1,70 @@
+﻿using OpenAI.Models;
+using System.ClientModel.Primitives;
+using System.Diagnostics;
+using System.Text.Json;
+
+namespace AzureOpenAI.Models;
+
+internal class AzureCreateChatCompletionRequest : IJsonModel<AzureCreateChatCompletionRequest>
```

Comment:
Internal model wraps third-party model to enable serializing model with Azure-added properties.